test/jdk/javax/swing/JFileChooser/FileSizeCheck.java

```diff
@@ -78,30 +83,30 @@ public static void test() {
         } catch (IOException ex) {
             throw new RuntimeException(ex);
         }
+        frame.add(fc);
+        frame.pack();
+        frame.setVisible(true);
+        frame.setDefaultCloseOperation(WindowConstants.EXIT_ON_CLOSE);
+    }
 
-        fc.showOpenDialog(null);
-
-        // delete temp files
+    public static void main(String args[]) throws InterruptedException, InvocationTargetException {
+        passFailJFrame = new PassFailJFrame("JFileChooser Test Instructions" , INSTRUCTIONS, 5, 19, 35);
         try {
-            for (int i = 0 ; i < tempFilePaths.length ; ++i) {
-                Files.deleteIfExists(Paths.get(tempFilePaths[i].toString()));
+            SwingUtilities.invokeAndWait(new Runnable() {
+                public void run() {
+                    test();
+                }
+            });
```

Comment:
I suggest using method reference:

```suggestion
            SwingUtilities.invokeAndWait(FileSizeCheck::test);
```